examples/test_common.c

```diff
@@ -148,16 +150,47 @@ int write_checkpoint(int fd, int ckpt, char* buf, size_t size)
   return 1;
 }
 
+/* write the checkpoint data to shared fd, and return whether the write was successful */
+int write_shared_checkpoint(int fd, int ckpt, char* buf, size_t size, size_t offset)
```

Comment:
Can we drop this function since you've merged things to one call?

Reply:
oops.  I removed it from the header file but forgot..